library/src/G4CMPConfigMessenger.cc

```diff
@@ -185,7 +187,7 @@ G4CMPConfigMessenger::G4CMPConfigMessenger(G4CMPConfigManager* mgr)
 
   nielPartitionCmd = CreateCommand<G4UIcmdWithAString>("NIELPartition",
 	       "Select calculation for non-ionizing energy loss (NIEL)");
-  nielPartitionCmd->SetCandidates("Lindhard lindhard Lin lin LewinSmith lewinsmith Lewin lewin Lew Lew IMPACT Impact impact ImpactTunl impacttunl Sarkis sarkis Sar sar");
+  nielPartitionCmd->SetCandidates("Lindhard lindhard Lin lin LewinSmith lewinsmith Lewin lewin Lew Lew IMPACT Impact impact ImpactTunl impacttunl Sarkis sarkis Sar sar LindEmpirical EmpiricalLindhard EmpiricalLind");
```

Comment:
You cannot use "LindEmpirical" here; remove it.  Please add "Empirical", "empirical", "Emp" and "emp", consistently with the other name strings.  For ImpactTUNL, please add "Imp" and "imp" for consistency with your change in G4CMPConfigManager.